packs/datastax-cassandra/config.yml

```
@@ -16,3 +16,6 @@ documentation:
 keywords:
   - infrastructure
   - database
+installPlans:
+  - infra-agent-targeted
+  - cassandra-integration
```

This quickstart has a few things incorrect:

- It's not an on host integration
- It should be nested under the `java` quickstart
- The config needs to be reworked a bit to reflect that it's an APM agent install (fix `keywords`, etc.)

```suggestion
installPlans:
  - setup-java-agent
```